routemodel/data_retrieval/Time.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import os.path
+from routes import build_routes_points, format_routes_query,parse_routing_data
+from common import get_API_data
+from config import BASE_URL, API_KEY
+sys.path.append(os.path.dirname(__file__))
+
+def Time_to(speed: int,curLat: float, curLon: float, finLat: float, finLon: float ):
```

Comment:
snake case for method name please :))

Reply:
I also think this fits well enough into `routes.py` that it can just be another method there. @ufshk thoughts?

Reply:
could do that yes